iiblib/iib_build_details_pager.py

```diff
@@ -0,0 +1,54 @@
+from .iib_build_details_model import IIBBuildDetailsModel
```

Comment:
Maybe this pager could be in the iib_build_details_model.py as well. 
What do yo think?

Reply:
what is the reason why the model and the pager should be together?